Sources/UIViewExtensions.swift

```diff
@@ -31,8 +31,8 @@ extension UIView {
 
     /// EZSwiftExtensions, add multiple subviews
     public func addSubviews(_ views: [UIView]) {
-        views.forEach { eachView in
-            self.addSubview(eachView)
+        views.forEach { [weak self] eachView in
```

Comment:
Exactly. I was going to change it for a `for...in...` loop, but I decided against it, didn't want to change too much here